test/with_api_v1/mock.go

```diff
@@ -106,17 +107,18 @@ func (s *TestSilence) nativeSilence(opts *AcceptanceOpts) *types.Silence {
 	nsil := &types.Silence{}
 
 	for i := 0; i < len(s.match); i += 2 {
-		nsil.Matchers = append(nsil.Matchers, &types.Matcher{
+		nsil.Matchers = append(nsil.Matchers, &labels.Matcher{
+			Type:  labels.MatchEqual,
 			Name:  s.match[i],
 			Value: s.match[i+1],
 		})
 	}
 	for i := 0; i < len(s.matchRE); i += 2 {
-		nsil.Matchers = append(nsil.Matchers, &types.Matcher{
-			Name:    s.matchRE[i],
-			Value:   s.matchRE[i+1],
-			IsRegex: true,
-		})
+		m, err := labels.NewMatcher(labels.MatchRegexp, s.matchRE[i], s.matchRE[i+1])
```

Comment:
maybe rename "m"

Reply:
Given that the scope of this variable is 4 lines, I would claim that the name is fine. Open to suggestions, though.

Reply:
Yeah, I think it's fine as is.